.gitignore

```diff
@@ -39,3 +39,7 @@ updates.img
 vagrant/vagrant_data/*.iso
 vagrant/roles/subman-devel
 src/dnf-plugins/product-id/Makefile
+/bots
+/cockpit/subscription-manager-cockpit.tar.gz
+/node_modules
+/test
```

Comment:
Directory `test` should not be in .gitignore, because it includes files with unit tests. I wonder, why do you use absolute path starting with `/`?

Reply:
The leading `/` is there so that only the top-level `test` matches.  Just `test` would also match `./syspurpose/test`, for example.

`/test` specifically is in .gitignore because running the tests will create at least `./test/common` and `./test/images`, but I agree that ignoring all of `./test` is wrong.

I'll move all new files out of `./test` so we also don't need to ignore anything in there.